File: array/reverse.cpp
#include<iostream>
using namespace std;    
void reversearray( int arr[],int n){
int start=0;
int end=n-1;
while(start<end){
int temp=arr[start];
arr[start]=arr[end];
arr[end]=temp;
start++;
end--;
}


}
int main(){
int arr[] = {2,9,6,9,5,3};
int sz=6;
reversearray(arr,sz);
for(int i=0;i<sz;i++){
cout<<arr[i]<<" ";
}
return 0;


}

File: src/arrays/reverse.java
public class reverse {
    public static void reverse(int arr[]){
        int start=0;
        int end =arr.length-1;
        for (int i = 0; i < arr.length/2; i++) {
            int temp= arr[end];
            arr[end]=  arr[start];
            arr[start]= temp;
            start++;
            end--;
        }
    }
    public static void main(String[] args) {
        int arr[]={1,2,3,4,5,6,7};
        reverse(arr);
        for (int i = 0; i < arr.length; i++) {
            System.out.println(arr[i]);
        }
    }
}
